recipes/bluesky-darkframes/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "bluesky-darkframes" %}
+{% set version = "0.4.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_value = "0e8d2e4eefe74ea439630f53b36a4e6850c5dfa43a16e87f28745b5b57ece4b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: {{ hash_value }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky
+    - event-model >=1.14.0
+    - frozendict
+    - numpy
+
+test:
+  requires:
+    - caproto
+  commands:
+    - OPHYD_CONTROL_LAYER=caproto python -c 'import bluesky_darkframes'
```

Comment:
You will need to set this variable in a way Windows understands it. But do you need it at all? Would a simple import tests be enough here?